go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -412,15 +411,9 @@ func (cnci *CommonNetworkControllerInfo) UpdateNodeAnnotationWithRetry(nodeName
 // deleteNodeLogicalNetwork removes the logical switch and logical router port associated with the node
 func (bnc *BaseNetworkController) deleteNodeLogicalNetwork(nodeName string) error {
 	switchName := bnc.GetNetworkScopedName(nodeName)
-	// Remove switch to lb associations from the LBCache before removing the switch
```

Comment:
Heh, interesting you chose to remove this block instead of handling the errNotFound.
In a parallel PR where I am removing lb_cache this code block does indeed go away, but I am not sure if we want that in this PR.
Ref: https://github.com/ovn-org/ovn-kubernetes/pull/3387